pkg/manager/member/tikv_scaler.go

```diff
@@ -217,28 +216,6 @@ func (tsd *tikvScaler) SyncAutoScalerAnn(tc *v1alpha1.TidbCluster, actual *apps.
 		tc.Annotations[label.AnnTiKVAutoScalingOutOrdinals] = v
 		return nil
 	}
-
-	// For the auto-scaling slots, we would add the special hot region label to the store with pdapi.
-	pdClient := tsd.pdControl.GetPDClient(pdapi.Namespace(tc.Namespace), tc.Name, *tc.Spec.EnableTLSCluster)
-	for k := range currentScalingSlots {
-		podName := util.GetPodName(tc, v1alpha1.TiKVMemberType, k)
-		for _, store := range tc.Status.TiKV.Stores {
-			if store.PodName == podName {
-				id, err := strconv.ParseUint(store.ID, 10, 64)
-				if err != nil {
-					return err
-				}
-				ok, err := pdClient.SetStoreLabels(id, hostRegionLabel)
-				if err != nil {
-					return err
-				}
-				if !ok {
-					return fmt.Errorf("tc[%s/%s]'s pod[%s] failed to add special hot region label", tc.Namespace, tc.Name, podName)
-				}
-				break
-			}
-		}
-	}
 	return nil
 }
```

Comment:
previously we add the special region labels by using pdapi during tikv syncing process, which is successful to add the label we want, but the special scheduling need for the tikv pod created by auto-scaler can't be met. Thats' why we choose the starting args ways and it works. @aylei